schema_based_on_diagram.sql

```diff
@@ -0,0 +1,59 @@
+CREATE TABLE patients (
+    id  SERIAL PRIMARY KEY,
+    name varchar(100),
+    date_of_birth date
+)
+
+CREATE TABLE medical_histories (
+    id  SERIAL PRIMARY KEY,
+    admitted_at timestamp,
+    patient_id int,
+    status varchar(100)
+)
+
+CREATE TABLE invoices (
+    id  SERIAL PRIMARY KEY,
+    total_amount decimal(10,2),
+    generated_at timestamp,
+    payed_at timestamp,
+    medical_history_id int
+)
+
+CREATE TABLE treatments (
+    id  SERIAL PRIMARY KEY,
+    type varchar(100),
+    name varchar(100),
```

Comment:
- [ ] An error is thrown when creating the _treatments_ table due to the a syntax error on _line 25_. Let's correct this by removing the comma at the end of _line 25_.

![image](https://user-images.githubusercontent.com/12745474/222202167-ea2455a2-6110-4bea-88c1-13d9832276ee.png)

Reply:
Fixed.

Reply:
Done, as advised